Bottom: e98a34b71c4b83e7d191fbe9ac3bd36ce0bb716f
Top:    e0c238f79cebdf269b6052344ddf39e315b85648
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 02:18:58 +0000

main.c: Initialise crosshair unconditionally

Since we no longer generate graphics contexts from InitCrosshair(),
there is no need to check gui->gui before calling it.


---

diff --git a/src/main.c b/src/main.c
index ce95361..d3bbce3 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1988,8 +1988,7 @@ main (int argc, char *argv[])
 
   ResetStackAndVisibility ();
 
-  if (gui->gui)
-    InitCrosshair ();
+  InitCrosshair ();
   InitHandler ();
   InitBuffers ();
   SetMode (ARROW_MODE);
